lang/terraform/terraform.talon

```diff
@@ -0,0 +1,27 @@
+mode: user.code_terraform
+mode: user.auto_lang
+and code.language: terraform
+-
+tag(): user.code_comment_block_c_like
+tag(): user.code_comment_line
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_imperative
+tag(): user.code_operators_assignment
```

Comment:
> I could not implement most of the mutating definitions in tag `user.code_operators_assignment`. I included this tag only for the assignment operator.

Maybe this is a sign we need to split out the augmented assignment operators?

Reply:
Cc/ @wenkokke

Reply:
Probably? I just have a fully separate command for definitional equality in declarative languages.